tests/models.json

```diff
@@ -34,24 +38,21 @@
             "meta-llama/Llama-2-70b-chat-hf",
             "codellama/CodeLlama-34b-Instruct-hf",
             "mistralai/Mistral-7B-Instruct-v0.1"
+        ],
+        "image":[
```

Comment:
Can we remove these image models from anyscale. They are not supported

Reply:
Done